### DISCUSSION TITLE
Introductions (Hello World)

### DISCUSSION THREAD

**Discussion by bdougie** [floss-and-code/stripe-hackathon#4] [General]
Take some time and introduce yourself and what you plan to work on. 

## Let us know:

- Where you from (Location)
- Where you work or if you are looking for work
- What is your favorite open source project on GitHub?

## Example

> Hi folks, I am Brian (bdougie), and looking to this event. I am a Developer Advocate and have been here for ~3years. I am based in Oakland, CA, and have an open-source project called [Open Sauced](https://github.com/open-sauced/open-sauced) that I have been working on for a few years.

> My favorite open source project right now is https://github.com/vercel/next.js. I currently have my portfolio site, [bdougie.live](https://bdougie.live/) using it.

**Comment by ericalloyd** [comment]
👋 Hey everyone, I'm Erica. I'm based out of Austin, TX and work remotely for GitHub so I can be anywhere in the world doing what I love to do. The first open-sourced project that comes to mind for me is [ruby](https://github.com/ruby/ruby). The community is fun and embraced me when I was starting out as a developer, so I'll always have ❤️ for it.

Have fun hacking!

**Reply by thakopian** [reply]
Thanks for the training prep yesterday @ericalloyd !

**Comment by gallexi** [comment]
Hello! My name is Lexi, and I'm based out of the San Francisco area, CA. I too work remotely for GitHub, and I love that I can work from anywhere (in non-pandemic times =] ).

I'm newer to OSS, but I think [Open Collective](https://github.com/opencollective) is pretty cool! I contributed by opening some issues when I noticed a privacy feature was making things _too_ private, and not showing me my own information 😅 They were super cool about it, and really friendly!

**Reply by bdougie** [reply]
Wow! But also cool to see you could contribute to the security of all users using open source. I am personally looking forward to hearing more about security on GitHub next at the [GitHub Universe](https://githubuniverse.com/) event 😄

**Comment by kmcq** [comment]
Hi! I'm Keith and I live in New York City 🏙️ I work at GitHub as well :octocat: on one of our security teams.

In my spare time I enjoy helping my artist friends have a web presence by building them websites and I've loved how I can get up and running quickly using [Next.js](https://github.com/vercel/next.js).

A little more fun, I love https://github.com/khanhas/spicetify-cli for making my Spotify app look really cool 😸

**Reply by bdougie** [reply]
@kmcq Love that you are here and love Next.js. I am only a recent user within the last 6 months but love it.

**Comment by osha7** [comment]
Hi everyone!  I'm Osha.  From NYC => Currently in Denver. I recently graduated from The Flatiron School - Full Stack Bootcamp program and am looking 👀 for my first Front End role now, while trying to learn algorithms.  

This is my first time with open source, and a hackathon - v. nervous + excited!

**Reply by bdougie** [reply]
Congrats on graduating from Flat Iron @osha7 and glad you are here!

**Reply by ericalloyd** [reply]
Congrats @osha7 🎉 First bootcamp, now a hackathon 💪 🏆 Welcome!

**Reply by osha7** [reply]
Thanks @bdougie !  I've heard such great things about you from Sam Miyakawa (Junior Developers Happy Hour) and Jessie Wu.  When I saw you were working on this event...I thought I should def give it a try!!!

@ericalloyd Thank you so much for such a great info session yesterday!  There was so much info included I've been hoping to learn lately, but didn't know where to go exactly.

**Comment by arelia** [comment]
Hi 👋🏾   I'm @arelia and I'm based in Chicago. I'm the developer program manager for teachers on the [GitHub Education](https://education.github.com) team, so I manage our Campus Advisors program and help make sure teachers have real-world tools to use in their classrooms. My favorite open source project is https://github.com/a11yproject/a11yproject.com

**Comment by thakopian** [comment]
Hello Everyone I'm Tadeh! I want to learn more about the modern web dev environment and wanted to get knee deep with Github. I'm based out of Los Angeles and have been learning full stack coding. 

btw, is there a link to a live event for today's Floss and Code Zoom webinar or does it just start here?

**Reply by osha7** [reply]
https://www.youtube.com/watch?v=NvTWPdu35FM

**Reply by thakopian** [reply]
Unless of course it's this youtube link for today - https://www.youtube.com/watch?v=NvTWPdu35FM&ab_channel=GitHub

**Reply by thakopian** [reply]
lol @osha7  we posted at the same time! Thanks for the follow up.

**Reply by osha7** [reply]
🤣

**Comment by yusufalp** [comment]
Hi, my name is Yusuf and I live in SF Bay Area. I have completed a software engineering program from Thinkful focused on full-stack. I am so excited to be here today as this is a great opportunity to learn for me. Looking forward to collaborating with others.

**Reply by gallexi** [reply]
Congrats on completing the program!! Welcome!

**Comment by muraterzioglu** [comment]
Hello everyone, @muraterzioglu here! My name is Murat, very jr. Full-stack developer, from Turkey I generally work with serverless and cloud technologies; writing in JavaScript, TypeScript and using Vue/React/Angular/Next/Nuxt and it goes on dependencies on what I need. 

Other information available on my profile, just check it and if you want the work with me on OpenSource projects or Startup just mail me! Let's increase our abilities together! 🥳 

Mail: [hello@muraterzioglu.dev](hello@muraterzioglu.dev)

**Reply by muraterzioglu** [reply]
p.n; I also work with databases and GCP and now, with floss and code, payment systems!

**Comment by ReginaF2012** [comment]
Hi everyone!

I'm Regina, I'm based out of the Greater St. Louis area. I'm a recent alum from Flatiron. @osha7 and I were in the same cohort. 😄 
I'm currently looking for my first software engineering role and this is my first hackathon.

I'm excited to do the this thing!

**Reply by gallexi** [reply]
Welcome Regina!! Congrats on your graduation from Flatiron!!
